### PR TITLE
ISSUE-1382 One vcard should support multiple addresses

### PR DESCRIPTION
**Pull request by quantranhong1999**
TODO debug fail tests in `OpenPaasContactsConsumerTest`.

```java
org.awaitility.core.ConditionTimeoutException: Assertion condition defined as a Lambda expression in com.linagora.tmail.contact.OpenPaasContactsConsumerTest [List check first element] 
expected: ContactFields(john@doe.com,John Dont,)
 but was: ContactFields(john@doe.com,John,Doe) within 10 seconds.
Expected :ContactFields(john@doe.com,John Dont,)
Actual   :ContactFields(john@doe.com,John,Doe)
```

### CODE REVIEW

File: tmail-backend/tmail-third-party/openpaas/src/test/java/com/linagora/tmail/contact/JCardObjectTest.java [hunk 1]
```diff
@@ -36,9 +40,9 @@ void shouldThrowIfFnOrEmailOptionalIsNull() {
             .isInstanceOf(NullPointerException.class);
     }
 
-    private Optional<MailAddress> maybeMailAddress(String email) {
+    private List<MailAddress> maybeMailAddress(String email) {
```
Comment (HoussemNasri) [inline tmail-backend/tmail-third-party/openpaas/src/test/java/com/linagora/tmail/contact/JCardObjectTest.java:43]: We should probably rename this method now

**Review by Arsnael — APPROVED** [review]

**Comment by Arsnael** [comment]
Related? https://james-jenkins.lin-saas.com/job/Tmail%20build/job/PR-1404/1/

**Comment by quantranhong1999** [comment]
> Related? https://james-jenkins.lin-saas.com/job/Tmail%20build/job/PR-1404/1/

Yes.

**Comment by quantranhong1999** [comment]
I did not have the time to continue this today. I will try to have a look tomorrow.

**Review by chibenwa — APPROVED** [review]

**Pull request merged**